cmd/govim/main_test.go

```diff
@@ -162,13 +156,45 @@ func TestInstallScripts(t *testing.T) {
 
 	govimPath := strings.TrimSpace(runCmd(t, "go", "list", "-m", "-f={{.Dir}}"))
 
+	// For the tests where we set GOVIM_USE_GOPLS_FROM_PATH=true, install
+	// gopls to a temp dir and add that dir to our PATH
+	td, err := installGoplsToTempDir()
+	if err != nil {
+		t.Fatalf("failed to install gopls to temp directory: %v", err)
+	}
+	defer os.RemoveAll(td)
+
 	t.Run("scripts", func(t *testing.T) {
 		testscript.Run(t, testscript.Params{
 			Dir: "testdatainstall",
 			Setup: func(e *testscript.Env) error {
 				e.Vars = append(e.Vars,
 					"PLUGIN_PATH="+govimPath,
-					"CURRENT_GOPATH="+os.Getenv("GOPATH"),
+					"CURRENT_GOPATH="+strings.TrimSpace(runCmd(t, "go", "env", "GOPATH")),
+					testsetup.EnvLoadTestAPI+"=true",
+				)
+				return nil
+			},
+		})
+	})
+
+	t.Run("scripts-with-gopls-from-path", func(t *testing.T) {
+		testscript.Run(t, testscript.Params{
+			Dir: "testdatainstall",
+			Setup: func(e *testscript.Env) error {
+				var currPath string
+				for i := len(e.Vars) - 1; i >= 0; i-- {
+					v := e.Vars[i]
+					if strings.HasPrefix(v, "PATH=") {
+						currPath = v
+						break
+					}
+				}
+				e.Vars = append(e.Vars,
```

Comment:
Is it important for this test to prepend this path segment? A simpler alternative would be to just append `PATH=td` and blow away what was there.

Reply:
Per previous comment, this is really to avoid relying on the fact that `PATH` contains nothing and should only contain `td`